docs/SETUP.md

```diff
@@ -84,43 +74,57 @@ Once installed, it should be available on your PATH: running `carton -v` should
 
 ### Install Dependencies
 
-# clone the Scrollery-website repository (if not already done)
+#### Clone the Scrollery-website repository (if not already done)
 
 ```bash
 git clone https://github.com/Scripta-Qumranica-Electronica/Scrollery-website.git
 ```
 
+#### Install website dependencies
+
 From the root of this repository `./Scrollery-website`, run the following command to locally install all npm dependencies:
 
 ```bash
 yarn --pure-lockfile
 ```
 
-From the `resources/cgi-bin` folder of this repository, run the following command to locally install all perl dependencies:
+#### Install SQE_API
+
+From the root of this repository `./Scrollery-website`, install the SQE API to `./Scrollery-website/resources/perl-libs`:
+
+```bash
+git clone https://github.com/Scripta-Qumranica-Electronica/SQE_DB_API resources/perl-libs
+```
+
+By default, the library is configured to connect to the Docker container automatically. However, if you need, you can modify the file `./resources/perl-libs/SQE_Restricted.pm` with your custom database credentials. Consult the [documentation for that repository](https://github.com/Scripta-Qumranica-Electronica/SQE_DB_API) for details on how to do this.
+
+#### Build the Perl cgi script dependencies
+
+From the `./Scrollery-website/resources/cgi-bin` folder of this repository, run the following command to locally install all perl dependencies:
 
 ```bash
 (sudo) carton install
 ```
 
-### Build Client-Side Code
+### Starting up the server for development or production
 
 You have a few options, depending on your workflow:
 
 ##### `npm start`
 
-Utilizes `webpack-hot-middleware` to achieve hot module reloading for all client-side assets. All files are watched and code rebuilt on changes; the changes will show up as soon as the Webpack build completes, without needing to refresh the browser.
+This is the easiest approach.  It utilizes `webpack-hot-middleware` to achieve hot module reloading for all client-side assets. All files are watched and code is rebuilt in realtime on changes; the changes will show up as soon as the Webpack rebuild completes, without needing to refresh the browser.
 
 Open up `http://localhost:9090` after running the command. This will run all requests to the Perl CGI scripts, bypassing your localhost (thus, it is not necessary to configure an Apache server!).
 
 ##### `npm run dev`
 
-In order to use this option, you must first configure an Apache server to serve the `index.html` file at the root of this project and is capable of serving Perl CGI scripts from `resources/cgi-bin`. If you don't want to this this, simply use the previous option instead (`npm start`).
+In order to use this option, you must first configure a webserver such as Apache to serve the `index.html` file at the root of this project and to execute the Perl CGI scripts from `resources/cgi-bin`. If you don't want to this this, simply use the previous option instead (`npm start`).
```

Comment:
I'm wondering if eventually we could throw together what the VirtualHost config looks like, and a similar configuration for NGINX. Just something to keep in mind if there's need.